src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -511,6 +511,21 @@
           Condition="'@(_InnerPackageCacheFiles)' != ''" />
   </Target>
 
+  <Target Name="CopyDotNetBuildArtifactsLogs"
+        Condition="'$(IsUtilityProject)' != 'true' and
+                    Exists('$(ProjectDirectory)artifacts')">
+
+    <ItemGroup>
+      <LogFilesToCopy Include="$(ProjectDirectory)artifacts/**/*.log" />
+      <LogFilesToCopy Include="$(ProjectDirectory)artifacts/**/*.binlog" />
+    </ItemGroup>
+    
+    <!-- Make a copy of the build logs -->
+    <Copy SourceFiles="@(LogFilesToCopy)"
+          DestinationFolder="$(DotNetBuildArtifactsLogDir)"
+          Condition="'@(LogFilesToCopy)' != ''" />  
```

Comment:
Given logs can be quite large, should we set UseSymboliclinksIfPossible=true ?

Reply:
We later clean origin so we would only be able to do that when `$(CleanWhileBuilding)` is not true.

Reply:
Overall, the long term strategy here is to not even need the copy and tell tooling to put any logs directly into the final destination.

Reply:
I agree with Alex here though. @Winniexu01 could this be a Move instead of a Copy (there's a Move msbuild task)?

Reply:
Updated, [R514-R526](https://github.com/dotnet/sdk/pull/42984/files#diff-0802dca0f4d533e4f44a76e021486aefa13bc1b193f018433b0d056ae84c9f3cR514-R526)